app/api/contact/contact.js

```diff
@@ -4,8 +4,9 @@ import settings from 'api/settings/settings';
 export default {
   async sendMessage({ email, name, message }) {
     const siteSettings = await settings.get();
+    const { senderEmail, siteName } = mailer.createSenderDetails(siteSettings);
     const mailOptions = {
-      from: '"Uwazi" <no-reply@uwazi.io>',
+      from: `"${siteName}" <${senderEmail}>`,
```

Comment:
@grafitto does it make sense that createSenderDetails returns the string already formed instead of the object ?

Reply:
Yes it does.. Changing that in a few